packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts

```diff
@@ -88,6 +88,17 @@ export interface IPersonaSharedProps extends React.HTMLAttributes<PersonaBase |
    */
   initialsColor?: PersonaInitialsColor | string;
 
+  /** The colors to be used for the presence-icon and it's background */
+  presenceColors?: {
+    presenceColorAvailable: string;
```

Comment:
Should we make the naming less redundant?

Reply:
Yes we should.